tests/winstanley/parsing/ParsingSpec.scala

```diff
@@ -0,0 +1,19 @@
+package winstanley.parsing
+
+import com.intellij.testFramework.ParsingTestCase
+import winstanley.WdlParserDefinition
+
+class ParsingSpec extends ParsingTestCase("", "wdl", new WdlParserDefinition()) {
+
+  // To add a new parsing test:
+  // - Add a new WDL file into testData (eg Hello.wdl)
+  // - Use PSI viewer to get a PSI breakdown of the file and copy into testData (eg Hello.txt)
+  //  - NB: you might need to do some PSI whitespace tidying up around the edges
+  // - Add a new test below (eg `def testHello(): Unit = doTest(true)`)
+
+  def testHello(): Unit = doTest(true)
```

Comment:
Does this test name need to correspond to the file names for the WDL file and the PSI breakdown text file? Asking for my own understanding.

Reply:
Correct. I'll comment it